snapcraft/plugins/catkin.py

```diff
@@ -34,6 +34,10 @@
     - include-roscore:
       (boolean)
       Whether or not to include roscore with the part. Defaults to true.
+    - rosinstall-files:
+      (list of strings)
+      List of rosinstall files to merge while pulling. Paths are relative to
+      the source.
```

Comment:
What do you think about writing a post in the forum or in your blog, to accompany this feature?
A place where we can point people to if they want to use it, with a nicely explained example?